attributes.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dd-agent-builder": {
+      "omnibus-ruby_branch": "datadog-5.5.0"
+  },
+  "omnibus": {
+     "build_user_password": "DDBuildR0cks!"
```

Comment:
Any way we could store this secret somewhere else and pass it through to the builder (ie. an env var or something?)